src/devices/src/virtio/net/tap.rs

```diff
@@ -66,6 +85,7 @@ fn build_terminated_if_name(if_name: &str) -> Result<[u8; IFACE_NAME_MAX_LEN]> {
     Ok(terminated_if_name)
 }
 
+#[derive(Copy, Clone)]
```

Comment:
Nit: I tend to favor deriving`Debug` on all types.

Reply:
I'd like to, but `ifreq` does not implement `Debug`, so can't derive